bitnami/keycloak/Chart.yaml

```
@@ -33,4 +33,4 @@ maintainers:
 name: keycloak
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/keycloak
-version: 24.0.4
+version: 24.0.5
```

We are adding a new feature, users can now change the metrics port, according to [semver](https://semever.org) we should bump the minor version:

```suggestion
version: 24.1.0
```